neo/rawio/axonarawio.py

```diff
@@ -521,27 +521,30 @@ def read_datetime(self):
         return datetime.datetime.strptime(date_str + ', ' + time_str,
                                           "%d %b %Y, %H:%M:%S")
 
-    def _get_channel_gain(self):
+    def _get_channel_gain(self, bytes_per_sample=2):
```

Comment:
Hi @sbuergers, that is good to know that the bytes_per_sample can vary. Are you sure this kind of information can not be extracted from one of the entries in the set file header?

Reply:
.bin files are always int16, .X files are always int8, according to the [FileFormatManual](http://space-memory-navigation.org/DacqUSBFileFormats.pdf). It might be burried somewhere in the `.set` file as well, but could not find it.